iothub/service/src/AmqpFileNotificationReceiver.cs

```diff
@@ -1,26 +1,27 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
+using System;
+using System.Threading.Tasks;
+using Microsoft.Azure.Amqp;
+using Microsoft.Azure.Devices.Common;
+using Microsoft.Azure.Devices.Common.Extensions;
+using Microsoft.Azure.Devices.Shared;
+
 namespace Microsoft.Azure.Devices
 {
-    using System;
-    using System.Threading.Tasks;
-    using Microsoft.Azure.Amqp;
-    using Microsoft.Azure.Devices.Common;
-    using Microsoft.Azure.Devices.Common.Extensions;
-
-    sealed class AmqpFileNotificationReceiver : FileNotificationReceiver<FileNotification>, IDisposable
+    internal sealed class AmqpFileNotificationReceiver : FileNotificationReceiver<FileNotification>, IDisposable
     {
         readonly FaultTolerantAmqpObject<ReceivingAmqpLink> faultTolerantReceivingLink;
-        readonly string receivingPath;
+        private readonly string _receivingPath;
 
         public AmqpFileNotificationReceiver(IotHubConnection iotHubConnection)
         {
-            this.Connection = iotHubConnection;
-            this.OpenTimeout = IotHubConnection.DefaultOpenTimeout;
-            this.OperationTimeout = IotHubConnection.DefaultOperationTimeout;
-            this.receivingPath = AmqpClientHelper.GetReceivingPath(EndpointKind.FileNotification);
-            this.faultTolerantReceivingLink = new FaultTolerantAmqpObject<ReceivingAmqpLink>(this.CreateReceivingLinkAsync, this.Connection.CloseLink);
+            Connection = iotHubConnection;
+            OpenTimeout = IotHubConnection.DefaultOpenTimeout;
+            OperationTimeout = IotHubConnection.DefaultOperationTimeout;
+            _receivingPath = AmqpClientHelper.GetReceivingPath(EndpointKind.FileNotification);
+            faultTolerantReceivingLink = new FaultTolerantAmqpObject<ReceivingAmqpLink>(CreateReceivingLinkAsync, Connection.CloseLink);
```

Comment:
seems to me faultTolerantReceiving link should also be renamed to include `_` at the beginning of the name

Reply:
you are correct, it is also missing its accessibility modifier. Added.